new-frontiers-14/frontier-station-14
# Ship category audit

## About the PR
Updated categories based on ship submission guidelines to be consistent. resolve #1962 

## Why / Balance
Categories for some ships were inconsistent with standards applied to other ships

Spreadsheet of current ship sizes and dimensions [here](https://docs.google.com/spreadsheets/d/1D4cii7fmX3AbP5dqDYnDnJTyUyPqvaF3/edit?usp=sharing&ouid=103554193349621686381&rtpof=true&sd=true)

## How to test
Go to each shipyard terminal
Verify ships show up in correct category
Buy and sell ships to ensure they do not break on load

## Media
<img width="494" alt="Ship Category - Exped Medium" src="https://github.com/user-attachments/assets/4f4e3265-647e-45cc-b5c5-ad97f05beafb">
<img width="479" alt="Ship Category - Frontier Medium" src="https://github.com/user-attachments/assets/7cf8489f-55d9-4028-801f-2887b276d4ba">
<img width="496" alt="Ship Category - Frontier Micro" src="https://github.com/user-attachments/assets/0355f84d-bfb7-426d-a29c-028617ed332f">
<img width="480" alt="Ship Category - Frontier Small" src="https://github.com/user-attachments/assets/2735aad9-7373-458d-b67d-f3756d60554f">
<img width="484" alt="Ship Category - Scrap Medium" src="https://github.com/user-attachments/assets/50e5906c-a56c-4b90-a978-5acefe3769ec">


## Requirements



## Breaking changes
N/A

**Changelog**
:cl:
- tweak: Updated category (size) for pathfinder, hammer, legman, pioneer, and orange
